codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -91,7 +95,11 @@ public void onStreamClosed(Http2Stream stream) {
                     FlowState state = state(stream);
                     int unconsumedBytes = state.unconsumedBytes();
                     if (ctx != null && unconsumedBytes > 0) {
-                        connectionState().consumeBytes(unconsumedBytes);
+                        if (!autoRefillConnectionWindow) {
```

Comment:
just use `this.autoRefillConnectionWindow`?  Does not using the `this.` version mean that the parameter has to be kept around longer than it otherwise would be?